migrations/sql/V2025.01.14.10.47__update_emli_to_ministry.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE mine_report_notification 
+    DROP CONSTRAINT mine_report_notification_contact_guid_fkey;
+
+ALTER TABLE emli_contact 
```

Comment:
I think we should hold off on renaming these tables / columns. They may be in use by the analytics teams (in Metabase or PowerBI), so might cause some issues

Reply:
that's a good point. I'll take a look after lunch